src/parser/statement.js

```diff
@@ -692,9 +693,13 @@ pp.parseClassBody = function (node) {
     if (!method.computed) {
       let { key } = method;
 
+      if (this.hasPlugin("flow") && this.isRelational("<")) {
+        isGeneric = true;
```

Comment:
Finding a good name is tricky though, so I'll just use `isClassMutatorStarter` for now